manifest.json

```diff
@@ -22,7 +22,7 @@
     },
     "multi_instance": true,
     "requirements": {
-        "yunohost": ">= 4.2.4"
+        "yunohost": ">= 4.3.0"
```

Comment:
I wonder if that would cause something like https://github.com/YunoHost-Apps/shaarli_ynh/pull/77#discussion_r767162490 😅

Then I would need to test it myself (or have someone install it) as Continuous Integration can't detect this.